improver_tests/spotdata/test_SpotLapseRateAdjust.py

```diff
@@ -309,8 +310,12 @@ def test_different_dimensions(self):
             spot_temperature_new_coord, self.neighbour_cube, self.lapse_rate_cube
         )
         expected = np.array([280 + (2 * DALR), 270, 280 - DALR]).astype(np.float32)
+        for slice in result.data:
+            self.assertArrayEqual(slice, expected)
 
-        self.assertArrayEqual(result[0].data, expected)
+    #    self.assertArrayEqual(result.data[0], expected)
+    #    self.assertArrayEqual(result.data[1], expected)
+    #    self.assertArrayEqual(result.data[2], expected)
```

Comment:
Please delete these commented lines

Reply:
Apologies, didn't notice this before pushing. Done.